src/Batch/Client/Tools/ObjectModelCodeGeneration/CodeGenerationLibrary/FileReader.cs

```diff
@@ -15,25 +15,30 @@
 ﻿namespace CodeGenerationLibrary
 {
     using System.IO;
+    using System.Linq;
     using Newtonsoft.Json;
 
     public class FileReader
     {
-        private readonly string fileName;
-        public FileReader(string fileName) { this.fileName = fileName; }
+        private readonly string folder;
+        private readonly string pattern;
+
+        public FileReader(string folder, string pattern)
```

Comment:
>public FileReader(string folder, string pattern) [](start = 8, length = 48)

This is breaking change, so either bump up the major version or add an overload to avoid version change.